From: Simon McVittie <smcv@debian.org>
Date: Fri, 11 Jul 2025 13:20:14 +0100
Subject: tests: Treat multithread-test as an Apache test

This test calls apache_init() to run Apache on a hard-coded port,
which means it cannot coexist with other tests in this group.
Don't allow it to parallelize with others.

Maybe helps: #1035983
---
 tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/meson.build b/tests/meson.build
index c13f676..615bdc1 100644
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -94,7 +94,6 @@ tests = [
   {'name': 'logger'},
   {'name': 'misc'},
   {'name': 'multipart'},
-  {'name': 'multithread'},
   {'name': 'no-ssl'},
   {'name': 'ntlm'},
   {'name': 'range', 'parallel': false},
@@ -149,6 +148,7 @@ if have_apache
   tests += [
     {'name': 'auth', 'parallel': false},
     {'name': 'connection', 'parallel': false},
+    {'name': 'multithread', 'parallel': false},
     {'name': 'proxy', 'parallel': false},
   ]
 
